src/cmd/input.go

```diff
@@ -44,10 +44,7 @@ func readInputConfig() {
 	if dataFile != "" {
 		if dataFile == "-" {
 			viper.SetConfigType("yaml")
-			if hasStdin() {
```

Comment:
do we need to remove the `hasStdin()` function too or is it still used somewhere?

Reply:
It's still used by this function to create a `check` object: https://github.com/OpsLevel/cli/blob/98ebc160da9a2a68319ec6b9caa717f37449c3ef/src/cmd/check.go#L63-L70

Which fans out to dozens of function calls to determine whether or not `promptui` should be used to ask the user for input, for example:
https://github.com/OpsLevel/cli/blob/98ebc160da9a2a68319ec6b9caa717f37449c3ef/src/common/prompts.go#L11-L36

If we wanted to remove the function (and `promptui` along with it) I think it's better for us to do that in another ticket, since the diff would be huge.